### PR TITLE
sntp: add page

### PR DESCRIPTION
**Pull request by taher-mosbah**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by kbdharun** [comment]
Welcome to tldr and thanks for your contribution @taher-mosbah. Please sign the CLA in the above comment for this PR to go through.

File: pages/osx/sntp.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# sntp
+
+> A very Simple Network Time Protocol client program.
+
+- Query SNTP server {{pool.ntp.org}} and display the time:
```
Comment (gutjuri) [inline pages/osx/sntp.md:5]:
```suggestion
- Query the SNTP server `pool.ntp.org` and display the time:
```

File: pages/osx/sntp.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+`sntp {{pool.ntp.org}}`
+
+- Synchrize the system clock with {{pool.ntp.org}} SNTP server:
```
Comment (gutjuri) [inline pages/osx/sntp.md:9]:
```suggestion
- Synchronize the system clock with the SNTP server at `pool.ntp.org`:
```

File: pages/osx/sntp.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+`sudo sntp -S {{pool.ntp.org}}`
+
+- Enabe debug logging:
```
Comment (gutjuri) [inline pages/osx/sntp.md:13]:
```suggestion
- Enable debug logging:
```

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thanks, I have added some suggestions.

**Comment by taher-mosbah** [comment]
Thanks for reviewing and sorry about the forced push, I signed CLA and implemented suggestions.

**Comment by gutjuri** [comment]
LGTM, thanks

**Review by gutjuri — APPROVED** [review]

File: pages/osx/sntp.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# sntp
+
+> A very Simple Network Time Protocol client program.
```
Comment (vitorhcl) [inline pages/osx/sntp.md:3]:
```suggestion
> A very Simple Network Time Protocol client program.
> More information: <https://keith.github.io/xcode-man-pages/sntp.1>.
```

**Review by vitorhcl — CHANGES_REQUESTED** [review]
LGTM! Consider adding the "More information" link as I suggested.

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution and welcome to tldr.

LGTM, after @vitorhcl's suggestion.

**Review by vitorhcl — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**